book/src/concepts/event-channel.md

```diff
@@ -0,0 +1,120 @@
+# Event Channel
+
+This chapter will be easier than the previous ones. 
+
+While it is not essential to understand it to use amethyst, it can make your life much much easier in a lot of situations where using only data would make your code too complex.
+
+## What is an event channel?
+
+An `EventChannel` acts like a queue for any type that implements `Event`.
+The `Event` type is automatically implemented for all types that are `Send`, `Sync` and have a static lifetime.
+
+It is a single producer/multiple receiver. This means that it should be used when you only have a single "thing" (usually a system) producing events.
+In most case, the `EventChannel` should be stored in a global resource for ease of access. More on this later.
+
+
+The only exception to this is when you have a **SINGLE** external thread producing the events and you want your game to have a read access over the `EventChannel`.
```

Comment:
I'd omit this line and the next.  This advice focuses on such a narrow use case it might confuse readers.